content/500-platform/10-about.mdx

```diff
@@ -42,6 +42,33 @@ In each workspace, you can:
 - manage billing, i.e. select a [subscription plan](https://www.prisma.io/pricing?utm_source=docs&utm_medium=platform-docs), configure payment methods, or view the invoice history.
 - view the usage of your enabled PDP products across all projects in that workspace.
 - invite other users to collaborate in the workspace.
+- access the [Optimize](/optimize) dashboard to measure query performance and receive AI-powered recommendations.
+
+#### Optimize
+
+Each workspace in your [Prisma Data Platform account](https://console.prisma.io/optimize) has an associated Optimize project.
```

Comment:
Just verify that "project" is the correct term. I feel like a session would be more correct.

Reply:
https://prisma-company.slack.com/archives/C07BUT3JKPW/p1725631620563389?thread_ts=1725547333.799519&cid=C07BUT3JKPW